src/blk/kernel/KernelDevice.cc

```diff
@@ -52,6 +52,11 @@ using ceph::make_timespan;
 using ceph::mono_clock;
 using ceph::operator <<;
 
+std::ostream& operator<<(std::ostream& os, const blk_access_mode_t buffered);
```

Comment:
this is confusing. the declaration of a function is located in a .cc file, while its implementation is located in anther .cc file.

Reply:
I think the declaration should go where the type is declared since they are closely related.